ledger-tool/src/main.rs

```diff
@@ -920,6 +921,31 @@ fn open_genesis_config_by(ledger_path: &Path, matches: &ArgMatches<'_>) -> Genes
     open_genesis_config(ledger_path, max_genesis_archive_unpacked_size)
 }
 
+/// Finds the accounts needed to replay slots `snapshot_slot` to `ending_slot`.
+/// Removes all other accounts from accounts_db, and updates the accounts hash
+/// and capitalization. This is used by the --minimize option in create-snapshot
+fn minimize_bank_for_snapshot(
+    blockstore: &Blockstore,
+    bank: Arc<Bank>,
```

Comment:
Probably can take &Arc<Bank> or even &Bank? This avoids a clone call on the callers.